src/main/java/com/lms/learning_management_system/entities/service/UserService.java

```diff
@@ -0,0 +1,49 @@
+package com.lms.learning_management_system.entities.service;
+
+import com.lms.learning_management_system.entities.UserEntity;
+import com.lms.learning_management_system.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
We use Lombok in the project.  Use @RequiredArgsConstructor, remove @Autowired annotation, and do userRepository final.

@Service
@RequiredArgsConstructor
public class UserService {

    private final UserRepository userRepository;